openedx/frontend-app-publisher
# feat: add course tags

Introducing course tags in the Publisher UI. Here is the link to the [ticket](https://2u-internal.atlassian.net/browse/PROD-2913)

Testing method:
1. Go to a course page and navigate to the **Topics** section in Course Details.
2. Add topics in format `tag1,tag2,tag3` and you should observe Tag chips appearing in the row above. Check _information_ helper for details.
3. You can click on any chip to remove the tag from the course.
4. Don't forget to Save the course at the end.

![Untitled design (1)](https://user-images.githubusercontent.com/56605828/190337721-089148bc-202a-407d-bec1-9a5d2643e169.gif)

The PR has a UI bug that does not allow searching for relevant tags. It will be fixed in [subsequent PR](https://2u-internal.atlassian.net/browse/PROD-2952)
